src/Advection/vector_invariant_cross_upwinding.jl

```diff
@@ -21,16 +21,19 @@
     @inbounds û = u[i, j, k]
     δ_stencil = scheme.upwinding.divergence_stencil
 
-    δᴿ = _biased_interpolate_xᶠᵃᵃ(i, j, k, grid, scheme, scheme.divergence_scheme, bias(û), flux_div_xyᶜᶜᶜ, δ_stencil, u, v) 
+    δᴿ  =    _biased_interpolate_xᶠᵃᵃ(i, j, k, grid, scheme, scheme.divergence_scheme, bias(û), flux_div_xyᶜᶜᶜ, δ_stencil, u, v) 
+    ∂ts = _symmetric_interpolate_xᶠᵃᵃ(i, j, k, grid, scheme, cross_scheme, V_times_∂t_∂s_grid)
```

Comment:
Is `∂t_∂s` right? Or is this just `∂t_s`? Note, this should be consistent everywhere. (I see a `∂ts` below.) We use the form `∂t_s` as a shorthand for `∂s/∂t` (the time-derivative of `s` with respect to `t`), elsewhere we have written `∂z_u` for example.

Reply:
ok make sense, I will switch everything to `∂t_s`. Then we can think about a better name for `s`. In MITgcm they use `hfac`. In NEMO they use `e3`.